src/renderer/components/Cards/ValidatorStat.tsx

```diff
@@ -33,20 +34,18 @@ const renderUptime = (props: IValidatorStatProps): React.ReactElement => {
     return(
         <div className="validator-card-container">
             <h5>{props.title}</h5>
-            <h1>{
-                props.value
-            }</h1>
-            <h5>DAYS</h5>
+            <h1>{props.value ? "Online" : "Offline"}</h1>
+            <h5>{props.type}</h5>
         </div>
     );
 };
 export const ValidatorStat: React.FunctionComponent<IValidatorStatProps> = (
     props: IValidatorStatProps) => {
-    
+
     switch(props.type){
         case "ROI": return renderROI(props);
         case "ETH": return renderBalance(props);
-        case "Uptime": return renderUptime(props);
+        case "Status": return renderUptime(props);
```

Comment:
?

Reply:
Uptime in design was by days so I guess this makes more sense?